### PR TITLE
Upgrade Tachyon to v0.4.0

### PR DESCRIPTION
**Pull request by haoyuan**

### CODE REVIEW

File: tachyon/setup.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 /root/spark-ec2/copy-dir /root/tachyon
 
-/root/tachyon/bin/format.sh
+/root/tachyon/bin/tachyon format
```
Comment (shivaram) [inline tachyon/setup.sh:5]: Was there a way to pick a tachyon version in spark_ec2.py ? If so we should do a switch case here as well and use the old command for 0.3 ?

Comment (haoyuan) [inline tachyon/setup.sh:5]: There is no TACHYON_VERSION in spark_ec2.py for now. I can add that after 0.4.0. And don't make users to launch 0.3.0 through the script.

Comment (shivaram) [inline tachyon/setup.sh:5]: Ok - Thanks for clarifying

**Comment by shivaram** [comment]
Merging this.

**Pull request merged**